projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -440,6 +440,23 @@ public static function is_gutenberg_available() {
 		return true;
 	}
 
+	/**
+	 * Check if the Newsletter feature should be enabled
+	 *
+	 * @since $$next_version$$
+	 *
+	 * @return bool
+	 */
+	public static function is_newsletter_enabled() {
+		require_once JETPACK__PLUGIN_DIR . '/modules/memberships/class-jetpack-memberships.php';
+		return (
+			apply_filters( 'jetpack_subscriptions_newsletter_feature_enabled', false )
+			&& class_exists( 'Jetpack_Memberships' )
+			// && Jetpack_Memberships::get_connected_account_id() - Returns False - TODO: Investigate
```

Comment:
`Jetpack_Memberships::get_connected_account_id()` appears to be flaky.

```
[03-Mar-2023 21:07:59 UTC] [63d1-23335 => public-api.wordpress.com/rest/v1.1/notifications/?http_envelope=1&fields=id%2Cnote_hash&number=20]
[03-Mar-2023 21:07:59 UTC] [63d1-23335] Jetpack_Memberships::get_connected_account_id():
[03-Mar-2023 21:07:59 UTC] [63d1-23335] FALSE
[03-Mar-2023 21:08:04 UTC] [4d85-23335 => romarioraffearn.wordpress.com/wp-admin/admin-ajax.php?frame-nonce=bdc7e543d3]
[03-Mar-2023 21:08:04 UTC] [4d85-23335] Jetpack_Memberships::get_connected_account_id():
[03-Mar-2023 21:08:04 UTC] [4d85-23335] 2789
[03-Mar-2023 21:08:29 UTC] [1cdc-23335 => public-api.wordpress.com/rest/v1.1/notifications/?http_envelope=1&fields=id%2Cnote_hash&number=20]
[03-Mar-2023 21:08:29 UTC] [1cdc-23335] Jetpack_Memberships::get_connected_account_id():
[03-Mar-2023 21:08:29 UTC] [1cdc-23335] FALSE
```

Reply:
`Jetpack_Memberships::get_blog_id()` appears to return the JP ID sometimes, which causes the `connected_account_id`  to be incorrect

Reply:
Separating the checks seems to have fixed this problem